### PR TITLE
Polish distributed XGBoost training on PAI

### PR DESCRIPTION
**Pull request by weiguoz**
1. Reduce parameter `num_workers` of function `dist_train`
2. Add `tracker.py`, because PAI(tf1.15) only supports Python2 and xgboost<=0.82 at this moment.

Note: Let's remove `tracker.py` if PAI supports xgboost>=1.0.0

### CODE REVIEW

**Review by shendiaomo — DISMISSED** [review]
LGTM++

File: python/sqlflow_submitter/xgboost/pai_rabit.py [hunk 1]
```diff
@@ -64,8 +64,7 @@ def __init__(self, host, nworkers, port):
                                 self.TrackerHelperHTTPRequestHandler)
         tracker_port = PaiXGBoostTracker.find_free_port()
         nslave = nworkers
-        tracker = xgb.RabitTracker(host, nslave, tracker_port,
-                                   tracker_port + 1)
+        tracker = RabitTracker(host, nslave, tracker_port, tracker_port + 1)
```
Comment (typhoonzero) [inline python/sqlflow_submitter/xgboost/pai_rabit.py:67]: Why still need to copy the `tracker.py` file? For PAI XGBoost install version reasons?

Comment (weiguoz) [reply python/sqlflow_submitter/xgboost/pai_rabit.py:67]: Yes.
I added a comment [here](https://github.com/sql-machine-learning/sqlflow/pull/2134#discussion_r422162527). Let's remove `tracker.py` when PAI tf115 supports Python3.

**Review by typhoonzero — APPROVED** [review]
LGTM

**Review by shendiaomo — APPROVED** [review]

**Pull request merged**